plugins/modules/mysql_user.py

```diff
@@ -145,7 +145,8 @@
     description:
       - Limit the user for certain server resources. Provided since MySQL 5.6 / MariaDB 10.2.
       - "Available options are C(MAX_QUERIES_PER_HOUR: num), C(MAX_UPDATES_PER_HOUR: num),
-        C(MAX_CONNECTIONS_PER_HOUR: num), C(MAX_USER_CONNECTIONS: num)."
+        C(MAX_CONNECTIONS_PER_HOUR: num), C(MAX_USER_CONNECTIONS: num), C(MAX_STATEMENT_TIME: num)."
+      - "Mariadb also supports the option C(MAX_STATEMENT_TIME: num) which is not supported by mysql."
```

Comment:
```suggestion
        C(MAX_CONNECTIONS_PER_HOUR: num), C(MAX_USER_CONNECTIONS: num), C(MAX_STATEMENT_TIME: num) (supported only for MariaDB since collection version 2.7.0)."
```

Reply:
I'm also OK to have it in a separate line as in your version (line 149). In this case, i would
- Remove the choice from line 148
- Add that it's supported since collection version `2.7.0`

Up to you

Reply:
I like yours better. Changed it to your version.

Reply:
thanks!